# Max and Min x, y points of cluster
geometry_msgs/Point cen
geometry_msgs/Point min
geometry_msgs/Point max
